provisioning.adoc

```diff
@@ -0,0 +1,303 @@
+// vim: tw=79
+= Provisioning with Tendrl
+
+:toc:
+
+== Layers of Provisioning
+
+. Bare-metal provisioning brings up the OS on the storage nodes in the specific
+network(s).
+. Tendrl provisioning sets up tendrl api, etcd and node agents on the
+administrator specified storage nodes. It is possible that this could be
+accomplished irrespective of tendrl’s provisioning modules, either by the bare
+metal system directly or as part of image based or template based deployments.
+
+At this point, tendrl itself is up and running and will have imported the host
+information such as cpu, ram, block storage information etc.
+
+Once tendrl is up and running, administrator either imports or creates a new
+cluster. The rest of the document discusses how these operations would be
+implemented via tendrl.
+
+
+== Components
+
+The following components are referred to in the rest of this document.
+
+Provisioner::
+This is the ansible based provisioning system that tendrl relies upon to
+install and upgrade the storage system. Tendrl takes care of setting up the
+necessary user accounts and ssh access to enable this system to function. This
```

Comment:
Yes, quite similar. The difference would be that tendrl would natively support this functionality rather than relying on the provisioning system to provide such a script.